services/movie_session.py

```diff
@@ -42,3 +42,10 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `delete_movie_session_by_id` function uses `.get()`, which can raise a `DoesNotExist` exception if no movie session with the given ID exists. Consider handling this exception to prevent runtime errors.